pkgs/development/python-modules/pydot/default.nix

```diff
@@ -1,23 +1,38 @@
 { lib
 , buildPythonPackage
 , fetchPypi
+, python
 , chardet
 , pyparsing
 , graphviz
 }:
 
 buildPythonPackage rec {
   pname = "pydot";
-  version = "1.4.0";
+  version = "1.4.1";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "02yp2k7p1kh0azwd932jhvfc3nxxdv9dimh7hdgwdnmp05yms6cq";
+    sha256 = "d49c9d4dd1913beec2a997f831543c8cbd53e535b1a739e921642fe416235f01";
   };
+
+  postPatch = ''
+    substituteInPlace pydot.py \
+      --replace "program_with_args = [program" "program_with_args = ['${graphviz}/bin/' + program" \
```

Comment:
Should I better use `os.path.join` here and in other places?

Reply:
From this line it's hard to see what's happening or how it should look like. Considering we don't support Windows, I don't think it matters much.